src/adagrams.js

```diff
@@ -1,15 +1,122 @@
+const LETTER_POOL = {
+  'A': 9, 
+  'B': 2, 
+  'C': 2, 
+  'D': 4, 
+  'E': 12, 
+  'F': 2, 
+  'G': 3, 
+  'H': 2, 
+  'I': 9, 
+  'J': 1, 
+  'K': 1, 
+  'L': 4, 
+  'M': 2, 
+  'N': 6, 
+  'O': 8, 
+  'P': 2, 
+  'Q': 1, 
+  'R': 6, 
+  'S': 4, 
+  'T': 6, 
+  'U': 4, 
+  'V': 2, 
+  'W': 2, 
+  'X': 1, 
+  'Y': 2, 
+  'Z': 1
+}
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
+
+  // Create a copy of LETTER_POOl
+  const copiedPool= { ...LETTER_POOL };
+
+  // Create an empty hand to place the 10 letters
+  let hand = []
+  while(hand.length < 10){
+    const keys = Object.keys(copiedPool); // Get an array of keys
+    const randomIndex = Math.floor(Math.random() * keys.length); // Generate a random index
+    let randomKey = keys[randomIndex]; // Get the key at the random index
```

Comment:
Right now you aren't actually considering the weights of each letter, how could you change your implementation to do such a thing?